autozone-poc/src/utlis/footer/Footer.js

```diff
@@ -9,11 +9,11 @@ const Footer = () => {
           <div className="row">
             <div className="col-12 col-md-6">
               <div className="single_widget_area d-flex mb-30">
-                <div className="footer-logo mr-50">
+                {/* <div className="footer-logo mr-50">
```

Comment:
Why you comment this line

Reply:
because logo2 img file was showing 404 error.